### ISSUE TITLE
Feature: Timer to start application and do recording

### ISSUE THREAD

**Issue by KejPi** [KejPi/AbracaDABra#113]

**Comment by andimik** [comment]
Timer is working, so you could set it to "solved" after you have merged it to the master branch.

I will also update the German translation.

**Comment by KejPi** [comment]
Please wait with the transaltions until is it somehow stable to avoid double work. I will probably do some changes, I am not satisfied with current implementation.

**Comment by andimik** [comment]
Understood

**Comment by KejPi** [comment]
And by the way - current implementation relies on DAB time. That means recording will not start if you lose DAB sync. This is just a warning in case you want to try to schedule recording something that may be important for you ;-)

**Comment by andimik** [comment]
Oh, good to know

**Comment by KejPi** [comment]
Details: 
The scheduled recording starts at -30 sec by countdown, then service selection at -10 sec and recording starts at -2 sec. Then application waits for end time and stops ongoing recording. 
All the time points are in DAB time (recalculated to local time offset). DAB time shall be updated every second according to a standard but in the application the resolution as roughly 1.5 sec that is why recording starts bit sooner. If DAB time is lost application keeps updating time once a minute so it shoud not happen that recording will never finish.

Do you think that this implementation is a major limitation?

**Comment by andimik** [comment]
Even an accuracy of 1 min is fine, as I always cut my recordings with https://github.com/mifi/lossless-cut

**Comment by KejPi** [comment]
OK, I will keep the implementation as it is, thanks for feedback! :-)

**Comment by andimik** [comment]
Regarding DAB time vs. PC time:

What about syncing it only once per mux, meaning when you connect to the Mux for the first time. This might prevent the sync loss problem you were talking about.

**Comment by KejPi** [comment]
FYI: I have checked the DAB time wrt. system time with the ensembles I am receiving and it differs for each ensemble and only one ensemble was aligned (+- 500ms). The difference between ensembles was in the range of -500ms to +3500ms but two week ago I saw one ensemble with more than 1 minute delay. Based on this experiment I think more reliable implementation would be with system time without any consideration od DAB time.

**Comment by andimik** [comment]
Yes, good idea. I don't want to rely on the DAB time. Have no idea if there are even ensembles with wrong time on air, but I know at least one DVB-T Slovene multiplex (several hours offset, no chance to use a Linux STB for recording on all multiplexes, as DVB time cannot be switched off) and many many RDS stations with time offset of several years (!).

**Comment by KejPi** [comment]
I have just pushed the new implementation, you can try if you want.

**Comment by KejPi** [comment]
@andimik Could you please update German translation? I think I will release the app as it is if you do not see any limitation of current schedule implementation. I have never used such feature so I am not sure if it is behaving expected way.

**Comment by andimik** [comment]
Done

**Comment by KejPi** [comment]
Thank you, I will merge it in the evening and make the official release hopefully.
